command/volume_create.go

```diff
@@ -106,69 +100,12 @@ func (c *VsmCreateCommand) Run(args []string) int {
 			return 1
 		}
 
-		resp := CreateAPIVsm(c.vsmname, c.size)
+		resp := mapiserver.CreateVolume(c.vsmname, c.size)
 		if resp != nil {
-			c.Ui.Error(fmt.Sprintf("Error Creating Volume %v", resp))
+			c.Ui.Error(fmt.Sprintf("Error Creating Volume: %v", resp))
```

Comment:
Does this not belong to old CLI framework ?

Reply:
this file still is not removed. will remove all the files under command/.. once the COBRA cli is functional

Reply:
ok